src/App.js

```diff
@@ -1,7 +1,24 @@
 import "./App.css";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
```

Comment:
특별한 이유가 없다면, BrowserRouter라는 명칭 그대로 가져와도 되겠네요.
```suggestion
import { BrowserRouter, Route, Routes } from "react-router-dom";
```